packages/dashboard/src/customization/widgets/table/emptyTableState.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, Button, SpaceBetween } from '@cloudscape-design/components';
+import type { FunctionComponent } from 'react';
+
+const EmptyTableState: FunctionComponent = () => {
```

Comment:
This is not a "state" so a better name would be "EmptyTable" or even "EmptyTableComponent"

Reply:
resolved